source/developers/quickstarts/setup_a_backend_dev_environment.rst

```diff
@@ -1,2 +1,278 @@
+
+These include:
+
+* Setting up your development environment;
+* Making your first commit;
+* Submitting your first pull request (PR);
+* Signing your Contributor License Agreement (CLA).
+
+We assume you are comfortable with the command line, and understand the basics
+of using Git, GitHub, and python basics.
+
+This Quickstart is written for first-time contributors to the Open edX project.
+If you are a plug-in developer or a frontend developer, please see our
+Quickstart guide for plug-in developers <link coming soon> or our Quickstart for
+frontend developers <link coming soon>.
+
+System Requirements
+*******************
+
+For the smoothest experience, we recommend that your computer has at least 16 GB
+of RAM, 2 CPUs, and at least 50 GB of Free disk space.
+
+Before you start
+****************
+
+This tutorial is written for users of Mac OS, either with an Intel or an Arm
+(Apple Silicone, M1, M2, etc.) processor. Additionally, you will need to have the following
+installed or configured, and know at least the basics of using them, before
+proceeding:
+
+* Git (see `GitHub's set up Git guide
+  <https://help.github.com/en/github/getting-started-with-github/set-up-git>`_)
+
+* A `GitHub account <https://github.com/signup>`_
+  * Additionally, we strongly recommend setting up 2-factor authentication
+
+* `Python 3.8 <https://www.python.org/downloads/>`_
+
+* Your favorite code editor (our team uses `VSCode
+  <https://code.visualstudio.com/download>`_, `Emacs
+  <https://emacsformacosx.com/>`_ or via `homebrew`_, `Vim
+  <https://github.com/macvim-dev/macvim>`_ or via `homebrew`_, and others)
+
+.. _homebrew: https://brew.sh
+
+Install Docker
+**************
+
+Install `Docker for Mac <https://docs.docker.com/docker-for-mac/>`_ and launch
+it. You can check that it is running correctly with::
+
+    docker run --rm hello-world
+
+You should see a message that starts with the following::
+
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+
+On macOS, by default, Docker allocates at most 2 GB of RAM to containers; the
+Open edX software requires at least 8 GB. You should follow these instructions
+from the `official Docker documentation
+<https://docs.docker.com/docker-for-mac/#advanced>`_ to allocate at least 8 GB
+to the Docker daemon.
+
+
+Installing Tutor Nightly
+************************
+
+Navigate into a folder to hold your Open edX repositories.
+We will assume you're using ``~/openedx``, but you can choose any folder.
+
+.. code-block:: bash
+
+  mkdir -p ~/openedx
+  cd ~/openedx
+
+Create a Python 3.8 virtual environment (we'll call it ``tutor-venv``) and activate it:
+
+.. code-block:: bash
+
+  python3.8 -m venv tutor-venv
+  . tutor-venv/bin/activate
+
+Install Tutor. This is a tool that helps you run Open edX.
+We will install the "Nightly" version of Tutor, which means it will run the latest
+version of Open edX (as opposed to the stable version of Open edX):
```

Comment:
```suggestion
version of the code (as opposed to the most recent named release):
```

Reply:
Specifically suggesting this because I think people would better understand "named release" over "stable version"